app/src/main/res/values/strings.xml

```diff
@@ -166,7 +166,7 @@
     <string name="pref_title_pebble_forceuntested">Enable untested features</string>
     <string name="pref_summary_pebble_forceuntested">Enable features that are untested. ENABLE ONLY IF YOU KNOW WHAT YOU ARE DOING!</string>
     <string name="pref_title_pebble_forcele">Always prefer BLE</string>
-    <string name="pref_summary_pebble_forcele">Use experimental Pebble LE support for all Pebbles instead of BT classic, requires paring a "Pebble LE" after non LE had been connected once</string>
+    <string name="pref_summary_pebble_forcele">Use experimental Pebble LE support for all Pebbles, instead of BT classic. This requires paring a "Pebble LE" after a non LE has been connected once</string>
```

Comment:
The "a" is actually wrong here.
Exactly the same Pebble has to be paired before. It just has two different Bluetooth names and the corresponding "non-LE" Pebble has to be paired and connected once before the "LE" Pebble (which has to be  physically the same device)